config/v1/types_apiserver.go

```diff
@@ -88,7 +99,15 @@ type Audit struct {
 	//
 	// If unset, the 'Default' profile is used as the default.
 	// +kubebuilder:default=Default
-	Profile AuditProfileType `json:"profile,omitempty"`
+	// customRules specify profiles per group. These profile take precedence over the
+	// top-level profile field if they apply. They are evaluation from top to bottom and
+	// the first one that matches, applies.
+	//
+	// +listType=map
+	// +listMapKey=group
+	// +optional
+	Profile     AuditProfileType `json:"profile,omitempty"`
```

Comment:
the profile field must go up. The go doc here is for CustomRules

Reply:
Will fix up

Reply:
I have a question on this, is there a case where we will need standalone `Profile`?
So the question is:
Do we need
A) Profile

    Profile
    Group

or
B) Profile
    Group

Reply:
The profile is always there. It defaults to `Default` if not explicitly set by the user.

So in the traditional case you only have the top-level profile. But with the new API you can add custom profiles per group.